examples/experimental/dagster-components/dagster_components_tests/stub_code_locations/sling_location/components/ingest/replication.yaml

```diff
@@ -0,0 +1,14 @@
+source: LOCAL
+target: DUCKDB
+
+defaults:
+  mode: full-refresh
+  object: "{stream_table}"
+
+streams:
+  <PLACEHOLDER>:
+    object: "main.tbl"
+    meta:
+      dagster:
```

Comment:
explicit choice being made here to use the replication.yaml as the source of truth for this asset key info. basic logic is that these streams are being set up explicitly in this yaml file, and so it's most convenient to put the dagster-specific configuration here as well, rather than trying to separately configure that in the `defs.yml`